dev/PushNotifications/PushNotificationsLongRunningTask/winmain.cpp

```diff
@@ -0,0 +1,7 @@
+﻿#include <windows.h>
+#include "pch.h"
+
+int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, PSTR /*lpCmdLine*/, int /*nCmdShow*/)
+{
+    return 0;
```

Comment:
Well now. That's an interesting app :P

Placeholder as you get the skeleton in place with implementation coming in a follow up PR?

Reply:
Yes. Implementation coming soon.